.github/workflows/release-periodically.yml

```diff
@@ -24,9 +24,24 @@ permissions:
   contents: write
 
 jobs:
+  check-date:
+    # run only on even weeks
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if the week number is even
+        id: check
+        run: |
+          week_no=$(date +"%V")
+          if [ $((10#${week_no}%2)) -eq 0 ]; then
+            echo "is_even=true" >> $GITHUB_OUTPUT
+          fi
+    outputs:
+      is_even: ${{ steps.check.outputs.is_even }}
+
   make-release-tag:
     # Don't run scheduled workflows on forks.
-    if: github.event_name != 'schedule' || github.repository == 'rhinstaller/anaconda'
+    needs: check-date
+    if: ${{ (github.event_name != 'schedule' || github.repository == 'rhinstaller/anaconda') && needs.check-date.outputs.is_even == 'true' }}
```

Comment:
Did you test the changes? I have a feeling that `== 'true'` doesn't work for boolean values and I had to fix these conditions the last time. Anyway, you can just check `needs.check-date.outputs.is_even` (unless I am missing something). See `steps.check-tag.outputs.can_bump_version` below.

Reply:
I guess the 'true' value is not taken as boolean but just a text. It's stored as true and compared to that?

Ideally we should test this.

Reply:
One additional note, if the running workflow was not scheduled, it means that it was triggered manually and it should be able to run regardless of the week parity.